src/components/Hero/Hero.stories.tsx

```diff
@@ -221,6 +237,9 @@ export const Tertiary: Story = {
         }
         heroType="tertiary"
         subHeaderText={otherSubHeaderText}
+        // foregroundColor="yellow"
```

Comment:
Can remove this.

Reply:
Ugh.  I did remove those lines, but I had to revert something and POOF, they were back in.